ccp/src/cu/proving_thread/mod.rs

```diff
@@ -21,6 +21,6 @@ pub(crate) mod sync;
 #[cfg(test)]
 mod tests;
 
-pub(crate) use async_::AsyncThreadError as ProvingThreadError;
 pub(crate) use async_::ProvingThreadAsync;
+pub(crate) use async_::ProvingThreadAsyncError as ProvingThreadError;
```

Comment:
What is the propose of this renaming?

Reply:
Because `ProvingThreadAsync` represent  `ProvingThread` as a whole for other part of CCP.